src/lexer.coffee

```diff
@@ -1178,6 +1178,9 @@ CSX_IDENTIFIER = /// ^
   ( (?: (?!\s)[\.\-$\w\x7f-\uffff] )+ ) # Like `IDENTIFIER`, but includes `-`s and `.`s.
 ///
 
+# Fragment: <></>
+CSX_FRAGMENT_IDENTIFIER = /// ^ ((?=>)) /// # Ends immediately with '>'.
```

Comment:
why not just `/// ^ () > ///`?

Reply:
I've tried that one before but didn't work.  At first, I thought I got it wrong and moved on to the different pattern.
But, now after your suggestion, I took a closer look and actually found a bug in the `lexer` where start and end CSX tag where compared. The code compares group match of the start tag with the full match of the end tag.
It accidentally worked since they are the same, but it failed on fragment tag since the full match is `>` and group match is ''.